src/styles/modules/button.scss

```diff
@@ -0,0 +1,12 @@
+.button{
```

Comment:
please, before pushing your changes check typos and spacings

Reply:
also if you remember you have two different buttons in your template, you can divide styles on `button-primary` and `button-secondary`, so you will have `<button class="button button-primary">`, `button` class will have all common button styles, and secondary or primary class will have distinctive styles.